app/views/catalog/_featured_items.html.erb

```diff
@@ -14,7 +14,7 @@
 								<%= content_tag(:span, "Browse #{digital_project[:name]} content", class: "sr-only", id: "#{digital_project_id_base}-search") -%>
 							<%- end -%>
 							<div class=" proj-jav btn-group d-flex" role="group">
-								<button class="btn btn-outline-secondary btn-sm" type="button" data-toggle="tooltip" aria-label="More Information" title="More Information" data-proj-more="#proj-<%=proj_counter%>" data-proj-title="<%= digital_project[:name] %>" onclick="return projectModal(this);"><span class="fa fa-info-circle" aria-hidden="true"></span></button>
+								<button class="btn btn-outline-secondary btn-sm" type="button" data-toggle="modal" aria-label="More Information" title="More Information" data-modal-body-delegate="#proj-<%=proj_counter%>" data-modal-title-delegate="#proj-<%= proj_counter %>-title" data-target="#dcvModal"><span class="fa fa-info-circle" aria-hidden="true"></span></button>
```

Comment:
tooltip removal on purpose?

Reply:
Ack, no - but there does have to be a change (the button has data-toggle=modal, but the span needs data-toggle=tooltip)

Reply:
That's what I did with the MODS XML widget in the item page, just need to do consistently